### ISSUE TITLE
PROFIEL:  ' wordt vertaald naar \' en zo krijg je Rue d\\\'Ixelles

### ISSUE THREAD

**Issue by vvim** [vvim/myrecy#4]
in page-profiel-ophaalpunt.php :

 ' wordt vertaald naar \' en zo krijg je Rue d\\'Ixelles , heeft dit iets te maken met character-escaping ?

**Comment by vvim** [comment]
StackOverflow answers:

http://stackoverflow.com/a/2573357/707700

http://stackoverflow.com/a/2573345/707700

and

http://stackoverflow.com/questions/11478669/php-mysql-best-use-and-practice-of-escaping-strings

**Comment by vvim** [comment]
We could test if it only occurs with the single quote ' . If so, then simply use a stringreplace ("\'","'") for all the strings in the form?

**Comment by vvim** [comment]
see commit a99ad810c175af7aa15d1288b173abdc3a3fc4b0

okay, I added `str_replace("\'", "'", $X)` for every `$X` . Will it interfere with the `htmlspecialchars()` ?

Changed original code:

```
<td><input type="text" name="naam_ophaalpunt" id="naam_ophaalpunt" value="<?php echo htmlspecialchars($ophaalpunt_from_db->naam); ?>" class="extra-long" /></td>
```

to

```
<td><input type="text" name="naam_ophaalpunt" id="naam_ophaalpunt" value="<?php echo htmlspecialchars(str_replace("\'", "'", $ophaalpunt_from_db->naam)); ?>" class="extra-long" /></td>
```

**Comment by vvim** [comment]
solved with commit a99ad81
